services/app-graphql/src/fragments/rateFieldsFragment.graphql

```diff
@@ -20,6 +20,19 @@ fragment rateFieldsFragment on Rate {
         ...updateInformationFields
     }
     status
-    parentContractID
```

Comment:
Question: is this an intentional change?

Reply:
Yes. The field is listed on line 8 so just removing a duplicate.